sql-select/inactive-customers.sql

```diff
@@ -0,0 +1,3 @@
+select *
+       from "customers"
+       where "isActive" = 'f';
```

Comment:
Since isActive is a boolean column, you should just use `true` or `false` without quotes:
`"isActive" = false`